RockLib.Messaging.SQS/DependencyInjection/SQSExtensions.cs

```diff
@@ -47,7 +66,7 @@ ISender CreateSQSSender(SQSSenderOptions options, IServiceProvider serviceProvid
         /// <returns>A builder allowing the sender to be decorated.</returns>
         public static ISenderBuilder AddSQSSender(this IServiceCollection services, string name,
             Action<SQSSenderOptions> configureOptions) =>
-            services.AddSQSSender(name, configureOptions, true);
+            services.AddSQSSender(name, configureOptions, true, ServiceLifetime.Singleton);
```

Comment:
Should we just use _defaultLifetime here too?

Reply:
yes, missed that one